src/Uns.ts

```diff
@@ -99,7 +99,7 @@ export default class Uns extends NamingService {
     this.unsl2 = new UnsInternal(
       UnsLocation.Layer2,
       source.locations.Layer2,
-      BlockchainType.MATIC,
+      BlockchainType.POL,
     );
```

Comment:
Looks like this is going to be `POL` for both Polygon and Base configs. It's probably fine as is, but might be worth passing the blockchain type into constructor args.